biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -2436,6 +2436,16 @@ public Parameters getParameters(String key, boolean allowDuplicates) {
 	}
 
 	public String system(boolean allowFail, String command, String input) throws IOException, InterruptedException {
+		return system(allowFail, getBase(), command, input);
+	}
+
+	public String system(boolean allowFail, String runDirectory, String command, String input)
+		throws IOException, InterruptedException, InterruptedException {
+		return system(allowFail, new File(runDirectory), command, input);
```

Comment:
Should this be IO.getFile(runDirectory) since that will handle some interesting cases like `~` expansion?